docs/guide/design.md

```diff
@@ -189,47 +189,45 @@ Instead of linking to stylesheets or embedding images like usually, you can use
 
 #### Controller
 
-While it is absolutely possible to template a whole website using only Twig, a custom PHP controller gives many more options of customizing the behavior of the pages.
+While it is possible to template a whole website using only Twig, a custom PHP controller gives many more options of customizing the behavior of the pages.
 
 See [Custom controllers](content_rendering.md#custom-controllers) for more information.
 
-## Usage
```

Comment:
Why do we have to delete the "Usage" title here?

Reply:
This is simply to better organization of the table of contents. If you preview this page with the changes made in https://github.com/ezsystems/developer-documentation/pull/72, the right-hand TOC is more readable without it.

Reply:
Level two headings will be in the sidebar, yes, the current build already has TOCs like this.